### ISSUE TITLE
如何测试 论文中的 time 和fps

### ISSUE THREAD

**Issue by anhuidinglieqiaodaima** [cfzd/Ultra-Fast-Lane-Detection-v2#156]
我在运行这行代码时遇到了一些问题 ：python speed_simple.py
(lane-det) root@autodl-container-ecb6119b52-5cbeb8ab:~/autodl-tmp# python speed_simple.py  
usage: speed_simple.py [-h] [--local_rank LOCAL_RANK] [--dataset DATASET]
                       [--data_root DATA_ROOT] [--epoch EPOCH]
                       [--batch_size BATCH_SIZE] [--optimizer OPTIMIZER]
                       [--learning_rate LEARNING_RATE]
                       [--weight_decay WEIGHT_DECAY] [--momentum MOMENTUM]
                       [--scheduler SCHEDULER] [--steps STEPS [STEPS ...]]
                       [--gamma GAMMA] [--warmup WARMUP]
                       [--warmup_iters WARMUP_ITERS] [--backbone BACKBONE]
                       [--griding_num GRIDING_NUM] [--use_aux USE_AUX]
                       [--sim_loss_w SIM_LOSS_W] [--shp_loss_w SHP_LOSS_W]
                       [--note NOTE] [--log_path LOG_PATH]
                       [--finetune FINETUNE] [--resume RESUME]
                       [--test_model TEST_MODEL]
                       [--test_work_dir TEST_WORK_DIR] [--num_lanes NUM_LANES]
                       [--auto_backup] [--var_loss_power VAR_LOSS_POWER]
                       [--num_row NUM_ROW] [--num_col NUM_COL]
                       [--train_width TRAIN_WIDTH]
                       [--train_height TRAIN_HEIGHT]
                       [--num_cell_row NUM_CELL_ROW]
                       [--num_cell_col NUM_CELL_COL]
                       [--mean_loss_w MEAN_LOSS_W] [--fc_norm FC_NORM]
                       [--soft_loss SOFT_LOSS]
                       [--cls_loss_col_w CLS_LOSS_COL_W]
                       [--cls_ext_col_w CLS_EXT_COL_W]
                       [--mean_loss_col_w MEAN_LOSS_COL_W]
                       [--eval_mode EVAL_MODE]
                       [--eval_during_training EVAL_DURING_TRAINING]
                       [--split_channel SPLIT_CHANNEL]
                       [--match_method {fixed,hungarian}]
                       [--selected_lane SELECTED_LANE [SELECTED_LANE ...]]
                       [--cumsum CUMSUM] [--masked MASKED]
                       config
speed_simple.py: error: the following arguments are required: config
 这与V1 不同   想知道原因

**Comment by zhouning1027** [comment]
请问一下，如何查看结果的fps呢？
